src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/CsvWriter.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.reports.ZipFileSummaryResponse;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public final class CsvWriter {
+
+    private CsvWriter() {
+        // utility class constructor
+    }
+
+    public static File writeZipFilesSumamryToCsv(
+        String fileName,
+        String[] csvHeader,
```

Comment:
`ZIP_FILES_SUMMARY_CSV_HEADERS` could be a constant in this class

Reply:
done